svc/FetchService.ts

```diff
@@ -314,6 +365,12 @@ export interface FetchOptions {
      */
     body?: any;
 
+    /**
+     * Unique identifier for this request, used for tracking and logging. If `true`, a UUID will be
+     * sourced from `LoadSpec.correlationId` or generated. If `false`, no correlationId will be set.
+     * */
```

Comment:
Don't think this is quite right - no defined behavior for true 

"Unique identifier for this request, used for tracking and logging. If unspecified, the service will check for a correlation ID on this config's `loadSpec` (if provided) or generate a new one if so enabled. If `false`, no correlationId will be set."

Exact wording / behavior might depend on our overall approach to enabling and originating these (see overall review comment)

Reply:
Thanks for flagging - we tweaked this behavior during CR (used to be as described in the docstring). Will update.